api/src/main/java/org/apache/gravitino/rel/TableChange.java

```diff
@@ -1353,6 +1353,9 @@ final class UpdateColumnPosition implements ColumnChange {
     private final ColumnPosition position;
 
     private UpdateColumnPosition(String[] fieldName, ColumnPosition position) {
+      if (position instanceof Default){
```

Comment:
please use Preconditions.checkArgument

Reply:
Hi @FANNG1 
I have use Preconditions.checkArgument for hanlding the argument. 
Thanks
Qiang